mlflow/pyfunc/__init__.py

```diff
@@ -724,6 +727,85 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def _download_model_conda_env(model_uri):
+    conda_yml_file_name = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)[ENV]
```

Comment:
```suggestion
    conda_yml_file_name = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)[ENV]
```

Does the `ENV` key already exist?

Reply:
Looks like it does.

Reply:
Not sure. What's the case ENV key does not exists ? In old version ?